### PR TITLE
Benefit groups

### PR DESCRIPTION
**Pull request by Oleh-Li**

### CODE REVIEW

**Comment by Oleh-Li** [comment]
@osavchukdoit  please check changes in last commit

File: src/components/homePage/BenefitGroups.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    height: "100%",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+
+  gradientWrapper: {
+    marginTop: 10,
+    marginBottom: 7,
+    height: 1,
+    backgroundColor: "rgba(76, 102, 159, 0.5)",
```
Comment (osavchukdoit) [inline src/components/homePage/BenefitGroups.js:116]: Lets extract colors to the THEME constant.

File: src/components/homePage/BenefitGroups.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+  benefitText: {
+    color: THEME.COLOR.GREY_DARK_TEXT,
+    fontFamily: FONTS.AVENIR.MEDIUM,
+    fontSize: 12,
+    lineHeight: 16,
+    marginBottom: 11,
+  },
+
+  rotateArrow: {
```
Comment (osavchukdoit) [inline src/components/homePage/BenefitGroups.js:135]: Looks like this property is not in use.

File: src/components/homePage/CostOverview.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+              "rgba(67, 190, 187, 0)",
+            ]}
+            start={{ y: 0, x: 1 }}
+            end={{ y: 0, x: 0 }}
+            style={styles.linearBackground}
+          />
+        </View>
+        <View style={styles.coverageTierWrapper}>
+          <View style={styles.titlesItemsWrapper}>
+            <Text style={[styles.titleItems, { width: "50%" }]}>
```
Comment (osavchukdoit) [inline src/components/homePage/CostOverview.js:54]: We should extract these "width" values to some common property to have an ability to change the values in one place for all columns. Can we skip the first column width value if we have "20" and "30" for two last items?

File: src/components/homePage/CostOverview.js [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+  linearBackground: {
+    position: "absolute",
+    left: 0,
+    right: 0,
+    top: 0,
+    height: "100%",
+  },
+
+  coverageTierWrapper: {},
```
Comment (osavchukdoit) [inline src/components/homePage/CostOverview.js:162]: Lets delete unused fields.

File: src/styles/theme.js [hunk 5]
```diff
@@ -27,5 +29,7 @@ export const THEME = {
     LOADER: "rgba(0,0,0,.5)",
     HEADER: "#1A3C5A",
     MENU: "#002D4F",
+    BUTTON_SHOW_HIDE_BG: "rgba(67, 190, 187, 0.1)",
```
Comment (osavchukdoit) [inline src/styles/theme.js:32]: Do we need the second "BUTTON_SHOW_HIDE_BG" const here?

**Review by osavchukdoit — APPROVED** [review]
Please add fixes according to the comments and don't forget to bump the version.

**Review by osavchukdoit — APPROVED** [review]
Able to merge

**Pull request merged**